src/plugins/intel_gpu/include/intel_gpu/primitives/eltwise.hpp

```diff
@@ -72,80 +72,92 @@ struct eltwise : public primitive_base<eltwise> {
           mode(mode),
           coefficients(std::vector<float>(0)),
-          stride(stride) {}
+          stride(stride),
+          broadcastSpec(spec.m_type, spec.m_axis) { }
 
     /// @brief Constructs eltwise primitive.
     /// @param id This primitive id.
     /// @param inputs Input primitives ids.
     /// @param mode Eltwise mode.
     /// @param data_type Expected output data type.
+    /// @param spec Auto broadcast rule specificiation.
     eltwise(const primitive_id& id,
             const std::vector<primitive_id>& inputs,
             eltwise_mode mode,
             data_types data_type,
+            const ov::op::AutoBroadcastSpec& spec = ov::op::AutoBroadcastSpec(ov::op::AutoBroadcastType::NUMPY),
             const padding& output_padding = padding())
         : primitive_base(id, inputs, output_padding, optional_data_type{data_type}),
           mode(mode),
           coefficients(std::vector<float>(0)),
-          stride(std::vector<tensor>(0)) {}
+          stride(std::vector<tensor>(0)),
+          broadcastSpec(spec.m_type, spec.m_axis) { }
 
     /// @brief Constructs eltwise primitive.
     /// @param id This primitive id.
     /// @param inputs Input primitives ids.
     /// @param mode Eltwise mode.
+    /// @param spec Auto broadcast rule specificiation.
     eltwise(const primitive_id& id,
             const std::vector<primitive_id>& inputs,
             eltwise_mode mode,
+            const ov::op::AutoBroadcastSpec& spec = ov::op::AutoBroadcastSpec(ov::op::AutoBroadcastType::NUMPY),
             const padding& output_padding = padding())
         : primitive_base(id, inputs, output_padding),
           mode(mode),
           coefficients(std::vector<float>(0)),
-          stride(std::vector<tensor>(0)) {}
+          stride(std::vector<tensor>(0)),
+          broadcastSpec(spec.m_type, spec.m_axis) { }
 
     /// @brief Constructs eltwise primitive.
     /// @param id This primitive id.
     /// @param inputs Input primitives ids.
-    /// @param coefficients Blob-wise coefficient for SUM operation
     /// @param mode Eltwise mode.
+    /// @param coefficients Blob-wise coefficient for SUM operation
+    /// @param data_type Expected output data type.
+    /// @param spec Auto broadcast rule specificiation.
     eltwise(const primitive_id& id,
             const std::vector<primitive_id>& inputs,
             eltwise_mode mode,
             const std::vector<float>& coefficients,
             data_types data_type,
+            const ov::op::AutoBroadcastSpec& spec = ov::op::AutoBroadcastSpec(ov::op::AutoBroadcastType::NUMPY),
             const padding& output_padding = padding())
         : primitive_base(id, inputs, output_padding, optional_data_type{data_type}),
           mode(mode),
           coefficients(coefficients),
-          stride(std::vector<tensor>(0)) {
+          stride(std::vector<tensor>(0)),
+          broadcastSpec(spec.m_type, spec.m_axis) {
```

Comment:
Cant we just assign the spec ? Doesnt it have copy constructor?

Reply:
Auto broadcast spec doesn't support copy construct now. Can I add it to auto broadcast struct?

Reply:
If it doesnt, no problem. No need to do.